README.md

```diff
@@ -21,46 +21,49 @@ SCALPEL can be installed and run using one of the following options:
 
 1. Clone the repository
 ```bash
-git clone https://github.com/p-CMRC-LAB/SCALPEL.git
-cd SCALPEL
+> git clone https://github.com/p-CMRC-LAB/SCALPEL.git
 ```
 
-2. Create the Conda environment
+2. Run SCALPEL using CONDA yml file
 ```bash
-conda env create -f requirements.yml
-conda activate scalpel_conda
+> nextflow run -resume SCALPEL/main.nf \
+  --sequencing chromium \
+  --samplesheet path/to/samplesheet.csv \
+  --transcriptome path/to/gencode.transcripts.fa \
+  --gtf path/to/gencode.annotation.gtf \
+  --ipdb path/to/mm10.polyA.track \
+  -with-conda SCALPEL/requirements.yml
```

Comment:
```suggestion
  --with-conda SCALPEL/requirements.yml
```

Or is this really just a single-dash option?

Reply:
I do the same in Snakemake. It's nice indeed. Just weird to see single-dash options with multiple letters.